workflow-manager/main.go

```diff
@@ -93,25 +181,66 @@ func main() {
 		log.Fatal(err)
 	}
 
-	var readyBatches []string
-	switch service {
+	var readyIngestionBatches []string
```

Comment:
Let's call these "intake" batches, to avoid the confusion I mentioned on Friday about how we don't actually do any ingestion; we do intake of batches produced by the ingestors.